.storybook/DocsDecorator/HealthFactors/index.styles.tsx

```diff
@@ -34,7 +34,7 @@ const styles = {
     css({
       width: `${pixelsToRem(24)}rem`,
       height: `${pixelsToRem(24)}rem`,
-      marginRight: theme.spacings.HALF,
+      marginRight: '0.5em',
```

Comment:
Should this be 'em'?

Reply:
fixed

Reply:
To be clear, this one has a value of 'em' just wondering why when the rest are 'rem.

Reply:
Webcore uses em to space their icons from the health factor text. I am assuming that their UX team wanted to scale the spacing in proportion of the health factors' font size. Not sure the reason of this choice, but I followed their design.